mod_reforged/hooks/config/strings.nut

```diff
@@ -743,9 +749,9 @@ local vanillaDescriptions = [
 					"Damage to [Hitpoints|Concept.Hitpoints] is reduced by " + ::MSU.Text.colorPositive("50%") + " and that to armor by " + ::MSU.Text.colorPositive("25%") + ".",
 					"The bonus drops exponentially when wearing head and body armor with a total penalty to [Maximum Fatigue|Concept.MaximumFatigue] above 15. The lighter your armor and helmet, the more you benefit.",
 					"Does not affect damage from mental attacks or status effects, but can help to avoid receiving them.",
-					"When attacking a target against whom you have a [Reach Disadvantage,|Concept.ReachAdvantage] reduce this disadvantage by " + ::MSU.Text.colorPositive(1) + " if your [Initiative|Concept.Initiative] is higher than that of your target."
+					"When attacking a target against whom you have a [Reach Disadvantage,|Concept.ReachAdvantage] reduce this disadvantage by " + ::MSU.Text.colorPositive(1) + " if your [Initiative|Concept.Initiative] is greater than that of your target."
 					"[Brawny|Perk+perk_brawny] does not affect this perk.",
-					"Cannot be picked if you have [Poise.|Perk+perk_rf_poise]"
+					"Cannot be taken if you have [Poise.|Perk+perk_rf_poise]"
```

Comment:
Unsure about this. Please explain why taken is better than picked.

Reply:
Flow and feels more refined.

Reply:
I don't feel that way. I may be wrong but my impression is that the community uses "pick" for perks a lot more than "take". In the vanilla tooltips themselves they use "pick", "learn" or "select" in the context of a perk. E.g.:
- Tip of the Day: `"Learn the \'Rotation\' or \'Footwork\' perks for additional mobility in battle."`
- Experience tooltip: `"Characters gain experience as they or their allies slay enemies in battles... ...he\'ll level up and be able to increase his attributes and pick a perk that grants a unique bonus..."`
- Levelup Tooltip: `"This character has leveled up. Increase his attributes and select a perk!"`

I don't like the use of `take` for perks.